modules/extension/tile-client/src/main/java/org/geotools/tile/util/TileLayer.java

```diff
@@ -153,6 +153,6 @@ private int calculateScale(ReferencedEnvelope extent, Rectangle screenArea) {
 
     private BufferedImage createImage(Rectangle rectangle) {
 
-        return new BufferedImage(rectangle.width, rectangle.height, BufferedImage.TYPE_INT_RGB);
+        return new BufferedImage(rectangle.width, rectangle.height, BufferedImage.TYPE_INT_ARGB);
```

Comment:
This is the substantive change that will need review by @aaime.

Reply:
@bencaradocdavies I actually did not even know this class existed (check the history, I did not work on this module), but yes, I've checked how it's used and believe the change is good.
@ianturton and @etj might want to provide a second opinion